### PR TITLE
feat(ProdCard1): added product card one

### PR DESCRIPTION
**Pull request by roshan-404**

### CODE REVIEW

File: client/shopping-app-clone/src/assets/scss/common/button/button.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
+@import "assets/scss/_variables.scss";
+
+.pri {
+  background: $primary-color;
+  color: $background-color;
+  border: 2px solid #46760a;
```
Comment (isVivek99) [inline client/shopping-app-clone/src/assets/scss/common/button/button.scss]: can we add a common color variable

File: client/shopping-app-clone/src/assets/scss/common/button/button.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+.sim {
+  background-color: $secondary-color;
+  color: #000;
+  border: 2px solid $secondary-color;
+}
+
+.str {
+  background-color: $background-color;
+  color: #000;
```
Comment (isVivek99) [inline client/shopping-app-clone/src/assets/scss/common/button/button.scss]: please add a common color variable

File: client/shopping-app-clone/src/components/common/button/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        {style.text}
+        {arrow === "ra" && (
+          <i className="fas fa-angle-right" aria-hidden="true"></i>
+        )}
+      </button>
+    </div>
+  );
+};
+
+export default Button;
```
Comment (isVivek99) [inline client/shopping-app-clone/src/components/common/button/Button.tsx:37]: please move the btn component to another file

File: client/shopping-app-clone/src/components/common/button/Button.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import React, { FC, useEffect, useState } from "react";
+import "assets/scss/common/button/button.scss";
+import "https://kit.fontawesome.com/637b14841d.js";
```
Comment (isVivek99) [inline client/shopping-app-clone/src/components/common/button/Button.tsx]: why do we need this, we have already imported font-awesome globally in index.html ?

File: client/shopping-app-clone/src/assets/scss/common/button/button.scss (hunk 1 above)
Comment (roshan-404) [reply client/shopping-app-clone/src/assets/scss/common/button/button.scss]: done

File: client/shopping-app-clone/src/assets/scss/common/button/button.scss (hunk 2 above)
Comment (roshan-404) [reply client/shopping-app-clone/src/assets/scss/common/button/button.scss]: done

File: client/shopping-app-clone/src/components/common/button/Button.tsx (hunk 3 above)
Comment (roshan-404) [reply client/shopping-app-clone/src/components/common/button/Button.tsx:37]: okay

File: client/shopping-app-clone/src/components/common/button/Button.tsx (hunk 4 above)
Comment (roshan-404) [reply client/shopping-app-clone/src/components/common/button/Button.tsx]: yeah removed

**Review by isVivek99 — APPROVED** [review]

**Pull request merged**